frontendUiTableImpl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/cell/BranchOrCommitCellRendererComponent.java

```diff
@@ -242,6 +250,19 @@ private static String getLabel(SyncToRemoteStatus status) {
         Case($(DivergedFromAndOlderThanRemote), "  (diverged from & older than ${remoteName})"));
   }
 
+  private static String getSyncToParentStatusBasedTooltip(IGitMacheteNonRootBranch currentBranch) {
```

Comment:
I wouldn't call it **current** branch since it incorrectly suggests that... well, it's a current branch ;) while it obviously doesn't need to be

Reply:
Also, `ToolTipText` maybe for clarity (since it's not really the tooltip object that's returned)